src/sentry/integrations/slack/utils/notifications.py

```diff
@@ -0,0 +1,173 @@
+import re
```

Comment:
@mgaeta Ignoring this file for review since I believe it's split from `utils.py`. If there are changes please let me know where.